### DISCUSSION TITLE
The name of the font

### DISCUSSION THREAD

**Discussion by psb1558** [psb1558/Junicode-font#187] [General]
Though I've got more requests in the queue, I'm going to freeze the project while I debug and tidy up; and when the font is tidy enough, I'll advance the version number to 2.0 and make a release.

I'll also drop the "Beta" from the family name, but I wanted to get a sense from anyone who reads this and cares to comment of whether the family name should be "Junicode" or "JunicodeTwo." "Junicode" is best for continuity, but "JunicodeTwo" lets one have the old version and the new installed side by side. I can't think of any other considerations.

As now, the variable version will have "VF" appended to the family name.

**Comment by alerque** [comment]
This topic makes me grumpy ... not at you but about the shortsightedness of early font format developers in creating a loose-loose situation for all font projects that put in the effort to keep developing after an initial release.

For my personal use of this project I'd much rather see the name stay the same and deal with the reflows myself, but I'm sure there will be folks who have good reasons not wish that work on themselves.

**Comment by kenmcd** [comment]
> Though I've got more requests in the queue, I'm going to freeze the project while I debug and tidy up; and when the font is tidy enough, I'll advance the version number to 2.0 and make a release.
> 

I have been meaning to post a number of issues found in v1.060 (when Italics were done).
Now I need to update those for v1.061 from yesterday.
My point is some of these issues really should be fixed before a non-beta release.
Been focusing on another project, but I will get these issues to you in the next day or two.
Will do some today.

FYI - my vote is for "Junicode" v2.000

**Comment by psb1558** [comment]
Two votes (so far) for plain old "Junicode."

@alerque: I agree. The font world is set up for commecial fonts that rarely see a new version after initial release. Open Source fonts often lead a different kind of life: this one has been in development on and off (mostly on) for more than thirty years, driven largely by feature requests from scholars.

@kenmcd: I am actively seeking out bugs and other problems, so issues will be very welcome. I will post frequent builds to help with the process.

**Comment by Doc73** [comment]
I agree with plain "Junicode"
